src/kitovu/utils.py

```diff
@@ -22,51 +23,49 @@ def get_password(plugin: str, identifier: str) -> str:
     return password
 
 
-class Error(Exception):
+class SchemaValidator:
+    """A validator for creating and merging errors in schema definitions."""
+
+    def __init__(self, abort: bool = True) -> None:
+        self.errors: typing.List[str] = []
+        self.abort: bool = abort
+
+    def validate(self, data: typing.Any, schema: typing.Dict[str, typing.Any]) -> None:
+        """Validates the given data with the schema."""
+        for error in jsonschema.Draft4Validator(schema).iter_errors(data):
+            self.errors.append(error)
+        if self.abort and not self.valid:
+            raise InvalidSettingsError(self)
+
+    @property
+    def valid(self) -> bool:
```

Comment:
`is_valid`?

Reply:
Alright, I'm used to the ruby version `valid?`